### DISCUSSION TITLE
Unable to edit meeting address

### DISCUSSION THREAD

**Discussion by shaggione** [code4recovery/12-step-meeting-list#1276] [Q&A] [answered]
Hello, I am seeking help as to why I cannot edit or add a meeting. I have reviewed previous discussions and, while I see similar posts, I don't see this specific problem.

I edited a meeting to change the meeting's address, and after clicking “update,” the address field went blank, and my change would not take. So I went to “add post” to add a new meeting, and after publishing it does not show up on the meeting list.

Our website is kerncountyaa.com. It is on Wordpress Version 6.4.1 and the version of the 12 Step Meeting List plug-in is 3.14.25.

We are using a Mapbox access token and Legacy UI.

I deactivated all the plugins that press would allow (some it won't let me), and, again, also tried to add the meeting as a new one. No success.
The only weird thing I saw in the console is 
rest-proxy/?v=2.0:16 WebSocket connection to 'wss://public-api.wordpress.com/pinghub/wpcom/me/newest-note-data' failed:
 Any suggestions?

Thank you,
Brian A.

**Comment by joshreisner** [comment]
hi @shaggione this sounds a lot like these two recent issues:

https://github.com/code4recovery/12-step-meeting-list/discussions/1273
https://github.com/code4recovery/12-step-meeting-list/discussions/1257

we haven't been able to reproduce this issue on our test site. where is your website hosted? aa-palmbeachcounty.org is hosted on wordpress.com and it sounds like devnasouthernutah.com might be as well

thanks

**Reply by joshreisner** [reply]
hi @shaggione we're researching this issue. it appears to be limited only to sites running on wordpress.com. we believe there may be some kind of dynamic database caching layer that wordpress.com added recently, which is preventing our plugin from seeing the connection between new meetings and their locations. we'll keep investigating and will update you when we know more.

**Comment by shaggione** [comment]
As well, we are hosted by Wordpress. An update problem?Thanks!Brian A.
    On Sunday, December 3, 2023 at 10:14:35 AM PST, Josh Reisner ***@***.***> wrote:  
 
 


hi @shaggione this sounds a lot like these two recent issues:

#1273
#1257

we haven't been able to reproduce this issue on our test site. where is your website hosted? aa-palmbeachcounty.org is hosted on wordpress.com and it sounds like devnasouthernutah.com might be as well

thanks

**Comment by shaggione** [comment]
I noticed something today. All these meetings changed address a couple of days ago.. The Wordpress meeting guide plugin meeting data was updated and all reverted to the old address. But no, 4 of the 5 changes are showing the new (correct) address. When you click on the Friday meeting, this shows.

This is the correct address. I'm wondering if, with some time, the previous page will show the updated address. Weird!F.Y.I.Brian A.
    On Monday, December 4, 2023 at 08:20:58 AM PST, Josh Reisner ***@***.***> wrote:  
 
 


hi @shaggione we're researching this issue. it appears to be limited only to sites running on wordpress.com. we believe there may be some kind of dynamic database caching layer that wordpress.com added recently, which is preventing our plugin from seeing the connection between new meetings and their locations. we'll keep investigating and will update you when we know more.

**Reply by joshreisner** [reply]
thanks @shaggione - you need to upload attachments at github.com - they won't come though in email. but yes this jibes with my understanding of the problem as well. when you make a change on wordpress.com the correct change is actually applied in the database right away, but then only later does that change become fully visible. i've seen it happen as quickly as 25 minutes, but sometimes much longer

**Comment by joshreisner** [comment, marked as answer]
hi @shaggione ok we have a change that we think will fix the issues on WordPress.com. if you're so inclined, you can[ read about it here](https://github.com/code4recovery/12-step-meeting-list/pull/1277), but the tl;dr is that since it's a big refactor we need to test it thoroughly before we roll it out to all users.

if you would like to help us test we would welcome that. you could download this ZIP file:  
[12-step-meeting-list.zip](https://github.com/code4recovery/12-step-meeting-list/files/13625711/12-step-meeting-list.zip) and then
on your site go to Plugins > Add New Plugin > Upload Plugin > Replace existing plugin

if you do that please let us know how it goes, but feel free not to and the update should roll out in a few days as a regular plugin update

**Comment by shaggione** [comment]
I have uploaded and installed the plug in. I will report on it's function.
    On Saturday, December 9, 2023 at 01:41:32 PM PST, Josh Reisner ***@***.***> wrote:  
 
 


hi @shaggione ok we have a change that we think will fix the issues on WordPress.com. if you're so inclined, you can read about it here, but the tl;dr is that since it's a big refactor we need to test it thoroughly before we roll it out to all users.

if you would like to help us test we would welcome that. you could download this ZIP file:
12-step-meeting-list.zip and then
on your site go to Plugins > Add New Plugin > Upload Plugin > Replace existing plugin

if you do that please let us know how it goes, but feel free not to and the update should roll out in a few days as a regular plugin update

**Reply by shaggione** [reply]
Plugin installed. I will report the outcome. Thank you!
